### DISCUSSION TITLE
filters improve subtitle hardcode

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#845] [Q&A] [answered]
What filters improve subtitle hardcode....

What field improve subtitle hardcode....

**Comment by jessielw** [comment, marked as answer]
Staxrip already has a filter for hardcoded subs and it works great. I'm not sure what you're asking here though?

**Reply by mr-qansuh** [reply]
I want to hardsub encoding without clear noise in video

**Reply by mr-qansuh** [reply]
> Staxrip already has a filter for hardcoded subs and it works great. I'm not sure what you're asking here though?

how can I edit a filter for hardcoded subs

**Reply by jessielw** [reply]
You can edit it in the avisynth or vipersynth filter panel on the lower left of the program. 

I'm not sure what you mean by visual noise, but I hope you figure out what you need. This really isn't an issue with staxrip, this is an external filter and I really dont know what you mean.

**Reply by mr-qansuh** [reply]
Thanks
